src/python/dxpy/scripts/dx.py

```diff
@@ -6002,6 +6003,21 @@ def positive_integer(value):
 parser_unarchive.set_defaults(func=archive, request_mode="unarchival")
 register_parser(parser_unarchive, categories='fs')
 
+#####################################
+# extract_dataset
+#####################################
+parser_extract_dataset = subparsers.add_parser('extract_dataset', help='(Preview feature) Retrieves the data, or generates SQL to retrieve the data, from a dataset or cohort for a set of entity.fields.',
+                                   description='(Preview feature) Retrieves the data, or generates SQL to retrieve the data, from a dataset or cohort for a set of entity.fields.',
+                                   prog='dx extract_dataset')
+parser_extract_dataset.add_argument('path', help='v3.0 Dataset or Cohort object ID (project-id:record-id where ":record-id" indicates the current selected project) or name')
```

Comment:
Shouldn't it say "where "record-id" indicates the currently selected record"?

Reply:
Updated to `v3.0 Dataset or Cohort object ID (project-id:record-id where ":record-id" indicates the record-id in current selected project) or name`

Thank you!